.github/workflows/protos.yml

```diff
@@ -46,6 +50,8 @@ jobs:
           sudo unzip $HOME/protoc.zip -d /tmp/protoc
           sudo chmod -R 777 /tmp/protoc
           echo "/tmp/protoc/bin" >> $GITHUB_PATH
+          pip install -r requirements/skinny-requirements.txt
+          pip install .
```

Comment:
@harupy any suggestion how to get the mlflow dependency properly installed? I don't wanna do `pip install mlflow` because that will not use the local version if I understand this correctly.

Now in the `proto` CI check I always get `mlflow not found` errors